testing/test_aux.sh

```diff
@@ -146,7 +146,7 @@ function redact {
         -e 's/[0-9]{4}-(0|1)[0-9]-(0|1|2|3)[0-9] [0-9]{2}:[0-9]{2}:[0-9]{2}\+00:00/XXX/' \
         -e 's/DAQ version.*//' \
         -e 's/[0-9].[0-9]{2} seconds/XXX/' \
-        -e 's/\b(?:\d{1,3}\.){3}\d{1,3}\b/XXX/'
+        -e 's/\b(\d{1,3}\.){3}\d{1,3}\b/XXX/'
```

Comment:
Actually this doesn't replace the ip in sed. Sed doesn't have the \d set, so you will have to expand the \d yourself into [0-9]. This is also why you're seeing the ip address diff in your test_aux run. Now it looked like test_aux wasn't running properly when the redact was invalid, so the diff you are seeing now was always there but it wasn't failing the aux test.  This should be fixed before merging in.